tensorboard/webapp/metrics/store/metrics_reducers_test.ts

```diff
@@ -362,10 +362,7 @@ describe('metrics reducers', () => {
         },
         cardList: [cardId1],
         cardToPinnedCopy: new Map([[cardId1, pinnedCopyId1]]),
-        cardToPinnedCopyCache: new Map([
-          [cardId1, pinnedCopyId1],
-          [cardId2, pinnedCopyId2],
-        ]),
+        cardToPinnedCopyCache: new Map([[cardId1, pinnedCopyId1]]),
```

Comment:
This does not seem in scope of this change but, I guess it is ok.

Reply:
so `cardToPinnedCopyCache` is designed to be aligned with any pin/unpin actions towards "cardToPinnedCopy".
Only when experiment removal (not the actual cards pin/unpin) this two maps could be different (and `cardToPinnedCopyCache` is a superset of `cardToPinnedCopy`)

this test checks "pinned cards removal", which is an unpin action, so we should also check here.
this bug should be caught earlier with this test only if I misunderstood removal of pinned cards to removal of experiments

Reply:
Correct myself here, this is _not_ unpin action but remove the cards from card list. The fix in in #5678